src/invidious/views/components/item.ecr

```diff
@@ -31,9 +31,13 @@
                 <% end %>
                 <p dir="auto"><%= HTML.escape(item.title) %></p>
             </a>
-            <a href="/channel/<%= item.ucid %>">
+            <% if !item.ucid.to_s.empty? %>
```

Comment:
I don't think that `.to_s` is required here.

Reply:
I get this compile error when removing the .to_s 
![image](https://github.com/iv-org/invidious/assets/78101139/365ea7f3-fa0b-4960-ab26-679c7d8f0f77)

Reply:
Ah, I see the problem. Urgh, this will need to be changed :c
For now, let's keep the `.to_s`, then.

FTR: it's caused by this bit of code in the `InvidiousPlaylist` struct:
https://github.com/iv-org/invidious/blob/507bed6313b49564e53b69a5c9b4d072d1e05e4b/src/invidious/playlists.cr#L229-L231